functions/Set-DbaSpn.ps1

```diff
@@ -80,7 +82,19 @@ Connects to Active Directory and adds a provided SPN to the given account. Uses
 			$serviceaccount = ($serviceaccount.split("@"))[0]
 		}
 		
-		$root = ([ADSI]"LDAP://RootDSE").defaultNamingContext
+		if ($domainName) {
+			$root = ""
+			$domainName = "boatmurder.drew.local"
```

Comment:
this wont work across the board ;) would $DomainName = $env:USERDNSDOMAIN work up there as a default value?